stripe/api_resources/abstract/updateable_api_resource.py

```diff
@@ -9,17 +9,16 @@ class UpdateableAPIResource(APIResource):
     @classmethod
     def modify(cls, sid, **params):
         url = "%s/%s" % (cls.class_url(), quote_plus(util.utf8(sid)))
-        return cls._static_request("post", url, **params)
```

Comment:
Sorry, what is the purpose of removing api_key support from the modify method?
How could I call modify without passing api_key explicitly?